pliers/extractors/text.py

```diff
@@ -402,3 +400,140 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+
+class PretrainedBertEncodingExtractor(ComplexTextExtractor):
+
+    ''' Uses transformers library to extract contextualized encodings for words
+    or text sequences using pre-trained BERT models.
+
+    Args:
+        pretrained_model_or_path (str): A string specifying which BERT
+            model to use. Can be one of pretrained BERT models listed at
+            https://huggingface.co/transformers/pretrained_models.html
+            or path to custom model.
+        tokenizer (str): Type of tokenization used in the tokenization step.
```

Comment:
What are the valid values here? If this is just passed through, make that explicit and point the reader to the place in the transformers docs that describes them. If it's a small set, probably best to just reproduce them here.

Reply:
I've specified that BERT models are those preceded by "bert" prefix in the list, it's a pretty long list and might be expanding in the future.